.github/workflows/sdk_build.yml

```diff
@@ -0,0 +1,41 @@
+name: Dapr dotnet sdk CI
+
+on: 
+  push:
+    branches:
+      - master
+      - release-*
+    tags:
+      - v*
+    
+  pull_request:
+    branches: 
+    - master
+    - release-*
+
+jobs:
+  build:
+    name: Build Projects
+    runs-on: windows-latest
+    strategy:
+      matrix:
+        configuration: [release, debug]
+    steps:
+      - uses: actions/checkout@v1
+      - name: Setup .NET Core
+        uses: actions/setup-dotnet@v1
+        with:
+          dotnet-version: 3.0.100
+      - name: Build solution - ${{ matrix.configuration }}
+        run: dotnet build code.sln --configuration ${{ matrix.configuration }}
+      - name: Test solution - ${{ matrix.configuration }}
+        run: dotnet test code.sln --configuration ${{ matrix.configuration }}
+      - name: Generate Nuget Packages - ${{ matrix.configuration }}
+        run: dotnet pack code.sln --configuration ${{ matrix.configuration }}
+      - run: mkdir -p bin\artifacts
+      - run: copy bin\${{ matrix.configuration }}\*.nupkg bin\artifacts
```

Comment:
It would be simpler to combine both steps to single step and add name:

```yaml
- name: Copy nuget to bin\artifacts
  run: |
    mkdir -p bin\artifacts
    copy bin\${{ matrix.configuration }}\*.nupkg bin\artifacts
```